src/core/stdc/stdlib.d

```diff
@@ -81,55 +81,55 @@ else version(CRuntime_Bionic) enum RAND_MAX = 0x7fffffff;
 else static assert( false, "Unsupported platform" );
 
 ///
-double  atof(in char* nptr);
+double  atof(scope const char* nptr);
 ///
-int     atoi(in char* nptr);
+int     atoi(scope const char* nptr);
 ///
-c_long  atol(in char* nptr);
+c_long  atol(scope const char* nptr);
 ///
-long    atoll(in char* nptr);
+long    atoll(scope const char* nptr);
```

Comment:
It was circulated for quite a while that `in` is supposed to mean `scope const`, in the compiler it was always just `const` but lots of people have used it as `scope const`. Have you thought about actually changing the semantic of `in`?